docs/application/native/guides/multimedia/sound-manager.md

```diff
@@ -181,6 +181,53 @@ To query sound device information:
           /* Failed to free the device list*/
       ```
 
+   5. More functionality for `SOUND_DEVICE_USB_AUDIO` with `SOUND_DEVICE_IO_DIRECTION_OUT` device.
+      When you detect a device with above type, you can utilize more functionality with the following functions:
```

Comment:
Please avoid using 'above' . You may like to rephrase the sentence as:

When you detect a device with mentioned type, you can utilize more functionality with the following functions:

Reply:
ok, done.